web/modules/weather_blocks/src/Plugin/Block/HourlyForecastBlock.php

```diff
@@ -21,54 +20,22 @@ class HourlyForecastBlock extends WeatherBlockBase
     /**
      * {@inheritdoc}
      */
-    public function blockForm($form, FormStateInterface $form_state)
-    {
-        $form = parent::blockForm($form, $form_state);
-        $config = $this->getConfiguration();
-        $max = $config["max_items"] ?? "12";
-
-        $form["max_items"] = [
-            "#type" => "textfield",
-            "#title" => "Maximum items to display",
-            "#default_value" => $max,
-        ];
-
-        return $form;
-    }
-
-    /**
-     * {@inheritdoc}
-     */
```

Comment:
I guess part of the reason we even had this configurable was for the testing pages, right? This is just me thinking aloud that we don't really use the testing pages so maybe should revisit similar code elsewhere?

Reply:
Originally we thought we'd configure the number of hours to display via the Drupal UI. While we were using Drupal's block layout, we technically could, but once we stopped using that, this config became meaningless. It was just cruft sitting around, so I took out the cruft.

We're also using block config for testing pages, where we can configure the blocks with specific WFO grid points. I think there ***could*** be a use there for state and county pages, and since we've already implemented it, we might as well leave it there for now. Once we get to those, then we'll know whether block configuration is actually useful for us.